.github/workflows/pre-commit.yml

```
@@ -23,8 +23,8 @@ jobs:
     uses: "redhat-cop/ansible_collections_tooling/.github/workflows/pre_commit_and_sanity.yml@main"
     with:
       collection_namespace: infra
-      collection_name: controller_configuration
-      collection_version: 2.10.0
+      collection_name: aap_configuration
+      collection_version: 3.0.0-devel
       collection_repo: https://github.com/redhat-cop/aap_configuration/
       collection_dependencies: awx.awx
```

can this take a list?

---

Not without changing it all I believe

---

We will need to make that change to fix the workflows

---

Ahh, I see what you're referring to now... (not clear in the GH ui). It does take a space separated list

```suggestion
      collection_dependencies: awx.awx ansible.eda ansible.hub
```

We would also need ansible.platform but that doesn't seem to be on galaxy so that may not be too helpful

---

@sean-m-sullivan I think you're involved with the ansible.platform collection... is it going to be on galaxy? (or am I missing it?)

---

I don't think so, we might need to change the workflow stuff but we can work on that later

---

It is not AFAIK, I was involved in the initial push of it, but have not been more recently, but it it is my understanding it will not be public.